packages/quantic/cypress/e2e/default-2/smart-snippet/smart-snippet.cypress.ts

```diff
@@ -141,5 +148,56 @@ describe('quantic-smart-snippet', () => {
         });
       });
     });
+
+    describe('when clicking the feedback like button', () => {
+      it('should properly log the analytics', () => {
+        visitPage();
+
+        scope('when clicking the like button', () => {
+          Expect.displaySmartSnippetCard(true);
+          Actions.clickSmartSnippetLikeButton();
+          Expect.logLikeSmartSnippet();
+          Expect.displayExplainWhyButton(false);
+        });
+      });
+    });
+
+    describe('when clicking the feedback dislike button', () => {
+      it('should properly log the analytics', () => {
+        visitPage();
+
+        scope('when clicking the dislike button', () => {
+          Expect.displaySmartSnippetCard(true);
+          Actions.clickSmartSnippetDislikeButton();
+          Expect.logDislikeSmartSnippet();
+          Expect.displayExplainWhyButton(true);
+        });
+
+        scope('when clicking the explain why button', () => {
+          Actions.clickSmartSnippetExplainWhyButton();
+          Expect.logOpenSmartSnippetFeedbackModal();
+        });
+
+        scope('when closing the feedback modal', () => {
+          Actions.clickFeedbackCancelButton();
+          Expect.logCloseSmartSnippetFeedbackModal();
+        });
+
+        scope('when selecting a feedback option', () => {
+          const exampleDetails = 'example details';
+          Actions.clickSmartSnippetExplainWhyButton();
+          Expect.logOpenSmartSnippetFeedbackModal();
+          Actions.clickFeedbackOption(feedbackOptions.length - 1);
```

Comment:
Since the `other` option is special, we could define a constant to make it clearer in the test.

```
const otherOption = 'other';
const feedbackOptions = [..., otherOption];
```

```suggestion
          Actions.clickFeedbackOption(otherOption);
```

Reply:
Sure thing I'll simplify this👌